modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -55,7 +55,7 @@ The DevWorkspace Operator functions as a standalone Operator and may be required
 ====
 
 .Procedure
-. Run the following commands to ensure that all `DevWorkspace` CRs are removed along with their related Kubernetes objects, such as deployments.
+. Remove `DevWorkspace` custom resources used by the operator, along with all related Kubernetes objects, such as deployments.
```

Comment:
There are two contexts in which the `DevWorkspace` phrase is used -- in one, we are referring to the operator itself, which is called `DevWorkspace Operator`. In the other, we are referring to the names of kubernetes objects, which derive from the operator name but are in lower-snake-case (e.g. `devworkspace-webhook-server`). 

I don't think it's a good idea to use title case for the latter examples, as trying to `oc get` that resource will fail, e.g. 
```
$ oc delete deployment/DevWorkspace-Webhook-Server -n openshift-operators
```
will return 
> Error from server (NotFound): deployments.apps "Devworkspace-Webhook-Server" not found

even if the deployment `devworkspace-webhook-server` is exists. Given the note we have that some resources may not exist on the cluster, this looks like a recipe for trouble.

Reply:
@amisevsk Throughout our docs we use 'Operator' in the title case only in sentences/content only. Whereas in code/commands blocks you can use 'Operator' in any case because that time is being used as an example.

Reply:
Hey @amisevsk  I think you misunderstood. 
I meant add the article 'the' before the word `DevWorkspace`
And wrt the case, in the word Operator, O needs to be capital throughout the PR. Right now in some places the pr has 'operator' we need to make it 'Operator'

Reply:
```suggestion
. Remove the `DevWorkspace` custom resources used by the Operator, along with all the related Kubernetes objects, such as deployments.
```

Reply:
I indeed did misunderstand, my apologies. I've updated all instances to `Operator` where it's used in a sentence. 

Regarding the last suggestion, I added `the DevWorkspace` and replaced `all the related` with `any related` -- "all the related" reads a little strangely to me.